include/geode/inspector/criterion/adjacency/brep_meshes_adjacency.h

```diff
@@ -26,19 +26,30 @@
 #include <absl/container/flat_hash_map.h>
 
 #include <geode/basic/pimpl.h>
+#include <geode/basic/uuid.h>
+
+#include <geode/mesh/core/solid_mesh.h>
+#include <geode/mesh/core/surface_mesh.h>
 
 #include <geode/inspector/common.h>
+#include <geode/inspector/information.h>
 
 namespace geode
 {
     class BRep;
-    struct uuid;
-    struct PolygonEdge;
-    struct PolyhedronFacet;
 } // namespace geode
 
 namespace geode
 {
+    struct BRepMeshesAdjacencyInspectionResult
+    {
+        InspectionIssues< uuid > meshes_with_wrong_adjacencies{ "" };
```

Comment:
Should we need a clear message?

Reply:
here and everywhere else?

Reply:
The message will be created in the inspection method